src/components/backend-ai-webui.ts

```diff
@@ -1152,6 +1158,7 @@ export default class BackendAIWebUI extends connect(store)(LitElement) {
               </address>
               <div id="sidebar-navbar-footer" class="vertical start end-justified layout" style="margin-left:16px;">
                 <backend-ai-help-button active style="margin-left:4px;"></backend-ai-help-button>
+                <mwc-icon-button class="download-link-menu-icon fg white" icon="download" slot="graphic" style="margin-left:4px;" @click="${() => this.showDownloadPage()}"></mwc-icon-button>
```

Comment:
It should be only shown on Web mode, not on App mode.

Reply:
reflected!